Bottom: 1f5b25ecd6a90e4be81ba9dfb4ad907df20e0b6c
Top:    f150652ca82a5866f82c590caaa5312f4a3a62bf
Author: Ian Kent <raven@themaw.net>
Date:   2013-03-26 09:12:36 +0800

autofs-5.0.7 - fix some automount(8) typos

Fix a couple of spelling errors in the automount(8) man page
that have been around for a long time.


---

diff --git a/CHANGELOG b/CHANGELOG
index d8e4049..488ad1e 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -37,6 +37,7 @@
 - fix libtirpc build option to require libtirpc-devel if needed.
 - fix systemd unidir in spec file.
 - document browse option in man page.
+- fix some automount(8) typos.
 
 25/07/2012 autofs-5.0.7
 =======================
diff --git a/man/automount.8 b/man/automount.8
index dddebce..7cc32c5 100644
--- a/man/automount.8
+++ b/man/automount.8
@@ -43,7 +43,7 @@ autofs managed mounts.
 .TP
 .I "\-d, \-\-debug"
 Enables logging of general status and progress messages as well as
-debuging messages for all autofs managed mounts.
+debugging messages for all autofs managed mounts.
 .TP
 .I "\-Dvariable=value"
 Define a global macro substitution variable. Global definitions
@@ -133,7 +133,7 @@ unused. If the force umount at exit option is used the filesystems
 will be umounted but the mount will not be released by the kernel
 until they are no longer in use by the processes that held them busy.
 If automount managed filesystems are found mounted when autofs is
-started they will be recoverd unless they are no longer present in
+started they will be recovered unless they are no longer present in
 the map in which case they need to umounted manually.
 .P
 If the option to disable the check to see if the daemon is already
